tools/localazy/checkForbiddenTerms.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import sys
+from xml.dom import minidom
+
+file = sys.argv[1]
+
+# Arrays of forbidden terms, with exceptions for some keys
+forbiddenTerms = [
+    [
+        "Element",
+        [
+            # Those 2 strings are only used in debug version
+            "screen_advanced_settings_element_call_base_url",
+            "screen_advanced_settings_element_call_base_url_description",
+            # only used for element.io homeserver, so it's fine
+            "screen_server_confirmation_message_login_element_dot_io",
+            # "Be in your element", will probably be changed on the forks, so we can ignore.
+            "screen_onboarding_welcome_title",
+        ]
```

Comment:
This format seems a bit weird, couldn't we get something similar with a dictionary?

Reply:
Yes, good idea: https://github.com/element-hq/element-x-android/pull/2686/commits/b0ff24347f76b96892a2bf221749ca685af51b00